http/stylesheets/_home.scss

```diff
@@ -1,24 +1,26 @@
 @use 'common/colors';
 
-h1#title {
-  text-align: center;
-  margin: 2rem 0 1rem;
-  font-size: 4rem;
-}
+#home-page {
+  h1#title {
+    text-align: center;
+    margin: 2rem 0 1rem;
+    font-size: 4rem;
+  }
 
-#text-container {
-  margin: 1rem auto;
-  padding: 1rem 3rem;
-  font-size: 1.2rem;
-  text-align: justify;
-  max-width: 800px;
+  #text-container {
+    margin: 1rem auto;
+    padding: 1rem 3rem;
+    font-size: 1.2rem;
+    text-align: justify;
+    max-width: 800px;
 
-  background: colors.$page-content-container-bg-color;
-  border: 1px solid colors.$nav-bg-color;
-  border-radius: 10px;
-  box-shadow: 1px 1px 6px colors.$faint-black-box-shadow-color;
+    background: colors.$page-content-container-bg-color;
+    border: 1px solid colors.$nav-bg-color;
+    border-radius: 10px;
+    box-shadow: 1px 1px 6px colors.$faint-black-box-shadow-color;
 
-  div {
-    margin: 1.5rem 0;
+    div {
+      margin: 1.5rem 0;
+    }
```

Comment:
is this just editor autoreformatting? should probably set up automatic linting somewhere so these type of commits don't get through, but blehhh 💤

Reply:
Oh, no, this is me putting all these styles within `#home-page` so that they don't accidentally override styles outside the homepage in the future. Nothing really changed except for adding `#home-page` at the top and indenting everything within